@aristath Remaining 2 problems on Level 0.

```
$ packages/bin/phpstan analyze -l 0
Note: Using configuration file /home/viktor/src/gridd/phpstan.neon.dist.
 117/117 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100%

 ------ ------------------------------------------------------
  Line   inc/classes/CSS.php
 ------ ------------------------------------------------------
  159    Access to an undefined property Gridd\CSS::$context.
 ------ ------------------------------------------------------

 ------ --------------------------------
  Line   template-parts/the-content.php
 ------ --------------------------------
  22     Function gridd not found.
 ------ --------------------------------


 [ERROR] Found 2 errors

```

--- a/composer.json
+++ b/composer.json
@@ -65,9 +65,6 @@
     "wptrt/wpthemereview": "*",
     "szepeviktor/phpstan-wordpress": "^0.6.0"
   },
-  "autoload": {
-    "classmap": ["inc/classes/", "grid-parts/"]
-  },
   "scripts": {
     "ci:syntax": "find . -type f -not -path './packages/*' -name '*.php' -print0|xargs -0 -P 4 -L 1 -- php -l",
     "ci:coding-standars": "phpcs --parallel=4",

